scripts/visualize/visualize_utils.R

```diff
@@ -209,40 +214,26 @@ locate_css_class_detail <- function(css, item_nm){
   return(item_details)
 }
 
-#' script to turn the dataviz into a thumbnail
-#' 
-visualize.map_thumbnail <- function(viz){
+createHurricaneSnapshot <- function(fig_height, fig_width, css, time_stamp, states, islands, 
```

Comment:
Nice refactor into functions. Would be helpful to add just a little documentation of the purpose of these for later readability.

Reply:
Good point! I will work on that